MSBot/src/msbot-connect-dispatch.ts

```diff
@@ -20,18 +21,18 @@ interface ConnectLuisArgs extends ILuisService {
 }
 
 program
-    .name("msbot connect dispatch")
+    .name('msbot connect dispatch')
```

Comment:
I recommend using double-quotes for strings that will need to be localized later.